components/brave_shields/core/browser/brave_shields_utils.h

```diff
@@ -117,13 +117,6 @@ void SetNoScriptControlType(HostContentSettingsMap* map,
 ControlType GetNoScriptControlType(HostContentSettingsMap* map,
                                    const GURL& url);
 
-void SetForgetFirstPartyStorageEnabled(HostContentSettingsMap* map,
-                                       bool is_enabled,
-                                       const GURL& url,
-                                       PrefService* local_state = nullptr);
-bool GetForgetFirstPartyStorageEnabled(HostContentSettingsMap* map,
-                                       const GURL& url);
-
 // Enables a webcompat exception for a specific URL.
 void SetWebcompatEnabled(HostContentSettingsMap* map,
                          ContentSettingsType webcompat_settings_type,
```

Comment:
Should these utilities also be moved?

Reply:
Sweet, is there an overall tracking issue for this? Its hard to tell that this is one of a chain just looking at this PR and the issue